uber/site_sections/registration.py

```diff
@@ -923,7 +923,7 @@ def stats(self):
         return json.dumps({
             'badges_sold': c.BADGES_SOLD,
             'remaining_badges': c.REMAINING_BADGES,
-
+            'badges_price': c.BADGE_PRICE,
```

Comment:
``badge_price`` would probably be better than ``badges_price`` but it's not a big deal either way.